mcs/class/System.ServiceModel/Test/System.ServiceModel/Bug36080Test.cs

```diff
@@ -82,15 +82,15 @@ public void Bug36080Test ()
 				client.SayHelloToAsync(str);
 
 				Assert.IsTrue (wait.WaitOne (TimeSpan.FromSeconds (20)), "timeout");
-#if MOBILE
+#if MOBILE && !XAMMAC_4_5
```

Comment:
This should be `|| XAMMAC_4_5` if you want the same behavior as mobile.

I believe this is the cause of the test failure you saw in https://github.com/xamarin/xamarin-macios/pull/3238#issuecomment-358434631

Reply:
Ah! thx!